.github/workflows/build.yml

```diff
@@ -4,18 +4,62 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-name: "wallet ci"
+name: "Wallet CI"
 
 env:
   GO_VERSION: 1.16
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
+  Prettier:
+    name: Prettier Check
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Run Wallet Web Prettier Check
+        run: |
+          echo '127.0.0.1 testnet.orb.local' | sudo tee -a /etc/hosts
+          echo '127.0.0.1 hub-router.trustbloc.local' | sudo tee -a /etc/hosts
+          echo '//npm.pkg.github.com/:_authToken=${GITHUB_TOKEN}' > ~/.npmrc
+          echo '@hyperledger:registry=https://npm.pkg.github.com' >> ~/.npmrc
+          echo '@trustbloc-cicd:registry=https://npm.pkg.github.com' >> ~/.npmrc
+          echo '@trustbloc:registry=https://npm.pkg.github.com' >> ~/.npmrc
+          make wallet-web-prettier-check
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  ESLint:
+    name: ESLint Check
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Go
```

Comment:
Setup Go?

Reply:
Can you elaborate, please? @sudeshrshetty

Reply:
I believe, @sudeshrshetty meant do we need this step ?

Reply:
I see, we probably don't - removed